great_expectations/core/batch_config.py

```diff
@@ -45,5 +45,10 @@ def build_batch_request(
             options=batch_request_options, partitioner=self.partitioner
         )
 
+    def batch_request_option_keys(self) -> tuple[str, ...]:
```

Comment:
Nit - should this be prefixed with `get_` for consistency?

Reply:
i'll remove this method entirely. it was meant as a user-facing replacement for the asset property `batch_request_options`, but let's evaluate if we actually want this, and then add it as a followup PR.

Reply:
removed in [9d42d57](https://github.com/great-expectations/great_expectations/pull/9499/commits/9d42d57dd3b7befff07071f70cdcdef8ef0de24d)